views/js/qtiItem/core/Loader.js

```diff
@@ -414,6 +414,10 @@ define([
         },
         loadPicData : function(pic, data){
             loadPortableCustomElementData(pic, data);
+        },
+
+        stripNamespaces: function (body) {
+           return body.replace(new RegExp('(".*?")|(?:[^\s\/<]+?:)', 'g'), '');
```

Comment:
Hmmm. I see an issue here, as the value in quotes will be removed as well as the namespaces. You should put a `$1` in the replace value. The regex contains one named expression `(".*?")` and one anonymous expression `(?:[^\s\/<]+?:)`  (because of the leading `(?:`). So the first expression will provide a parameter, while the second will be ignored. And to keep the value matched by the first expression you should provided the substitution `$1`.

BTW, instead of creating verbosly a regex, you could use the condensed form: `/(".*?")|(?:[^\s\/<]+?:)/g`, so the code should become:

```javascript
stripNamespaces: function stripNamespaces(body) {
    return body.replace(/(".*?")|(?:[^\s\/<]+?:)/g, '$1');
}
```

Reply:
I truly believe you. Not an expert in regex.
Tested looks stripped.